cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1048,7 +1052,11 @@ public MapAndStatus(T loadMap, K status) {
         String clusterNamespace = kafkaRebalance.getMetadata().getNamespace();
         if (clusterName == null) {
             LOGGER.warnCr(reconciliation, "Resource lacks label '{}': No cluster related to a possible rebalance.", Labels.STRIMZI_CLUSTER_LABEL);
-            return updateStatus(reconciliation, kafkaRebalance, new KafkaRebalanceStatus(),
+            KafkaRebalanceStatus status = new KafkaRebalanceStatus();
+            status.addCondition(StatusUtils.buildWarningCondition("Cluster Label missing", "Resource lacks label '"
```

Comment:
The string `Cluster Label missing` *has* to match one of the strings in the `onNew` method and if you change it you have to remember to change it in multiple locations. These condition strings should be move to variables or enum options so that there is a single source of truth for them.